modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -0,0 +1,236 @@
+package keeper_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+)
+
+func (suite *KeeperTestSuite) TestEscrowPacketFee() {
+	var (
+		err        error
+		refundAcc  sdk.AccAddress
+		ackFee     *sdk.Coin
+		receiveFee *sdk.Coin
+		timeoutFee *sdk.Coin
+	)
+
+	// refundAcc does not have balance for the following Coin
+	invalidCoin := &sdk.Coin{Denom: "cosmos", Amount: sdk.NewInt(8000000000)}
+	validChannelId := "channel-0"
+
+	testCases := []struct {
```

Comment:
Test cases where `RecvFee+AckFee > TimoutFee` and vice versa after implementing capital efficient escrow